subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -358,18 +238,14 @@ int nrf_cloud_decode_requested_state(const struct nrf_cloud_data *input,
 
 	const char *state_str = pairing_state_obj->valuestring;
 
-	if (compare(state_str, INITIATE_STR)) {
-		(*requested_state) = STATE_UA_INITIATE;
-	} else if (compare(state_str, PATTERN_WAIT_STR)) {
-		(*requested_state) = STATE_UA_INPUT_WAIT;
-	} else if (compare(state_str, PATTERN_MISMATCH_STR)) {
-		(*requested_state) = STATE_UA_INPUT_MISMATCH;
-	} else if (compare(state_str, TIMEOUT_STR)) {
-		(*requested_state) = STATE_UA_INPUT_TIMEOUT;
-	} else if (compare(state_str, PAIRED_STR)) {
-		(*requested_state) = STATE_UA_COMPLETE;
-	} else if (compare(state_str, DUA_PIN_STR)) {
+	if (compare(state_str, DUA_PIN_STR)) {
 		(*requested_state) = STATE_UA_PIN_WAIT;
+	} else {
+		LOG_ERR("Deprecated state. Delete device "
+			"from nrfCloud and "
+			"update device with JITP certificates.");
```

Comment:
Multiline prints break grepability, so for these, I'd say we need to either split them or break the 80 char limit